src/core/Codend.Application/Stories/Commands/UpdateStory/UpdateStoryCommandValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Codend.Application.Core.Errors;
+using Codend.Application.Extensions;
+using Codend.Domain.Core.Errors;
+using Codend.Domain.ValueObjects;
+using FluentValidation;
+using static Codend.Application.Core.Errors.ValidationErrors.Common;
+using static Codend.Domain.Core.Errors.DomainErrors.StringValueObject;
+
+namespace Codend.Application.Stories.Commands.UpdateStory;
+
+/// <summary>
+/// Validates <see cref="UpdateStoryCommand"/>.
+/// </summary>
+public class UpdateStoryCommandValidator : AbstractValidator<UpdateStoryCommand>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="UpdateStoryCommandValidator"/> class.
+    /// </summary>
+    public UpdateStoryCommandValidator()
```

Comment:
Imo there should be rule for epicId. If it should be updated then it shouldn't be empty